samples/bot-teams-authentication/python/appManifest/manifest.json

```diff
@@ -2,7 +2,8 @@
   "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.19/MicrosoftTeams.schema.json",
   "manifestVersion": "1.19",
```

Comment:
icons not needed for appManifest?

Reply:
Need FYI
![image](https://github.com/user-attachments/assets/5e6b6e2d-b1f8-458f-993e-2401113752f8)